PX4/Firmware
# VTOL: fix roll inversion

fixes #4797 

**Describe problem solved by the proposed pull request**
The VTOL attitude controller generates reversed fixed-wing roll commands. As a consequence, all VTOL mixers currently have a minus sign on roll actuators to compensate.

**Describe your preferred solution**
- Fix sign of the fixed wing commands generated by the VTOL attitude controller
- Fix builtin mixers
- Add a parameter to force users of custom VTOL mixers to check roll control surfaces before flight
- Fix builtin airframes scripts to bypass the new parameter

**Flight testing**
This requires extensive flight testing on all possible VTOL platforms.

Test card 1:
- Flash firmware on VTOL
- **Before flight**: Check direction of roll control surfaces
- Fly standard mission, the behaviour should be unchanged

Test card 2:
- set parameter V19_VT_ROLLDIR to 1
- try to arm
- the system should not generate actuator commands, error messages should be displayed on QGC.
- set parameter V19_VT_ROLLDIR to 0
- arm properly
